### PR TITLE
Create separate study view page documentation; Add simple filter description

### PR DESCRIPTION
**Pull request by zhx828**
To test
https://github.com/zhx828/cbioportal/blob/add-study-view-docs/docs/FAQ.md
https://github.com/zhx828/cbioportal/blob/add-study-view-docs/docs/SUMMARY.md

### CODE REVIEW

File: docs/study-view/simple-filter.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Simple Filter
+
+The study view support pre-filtered values of one attribute in URL.
+
+### How to
+Two parameters should be specified in the URL:
+
+* filterAttributeId: the clinical attribute id (we only accept one attribute id). Please refer to clinical_attribute_meta table, ATTR_ID column
```
Comment (jjgao) [inline docs/study-view/simple-filter.md]: We need to find a way to expose this. Maybe we can have it in the mouse over of chart header?

**Comment by jjgao** [comment]
I think we should address the issue about exposing attribute id on study view page - maybe in a separate PR.

I think we need a better place to put the documentation about how to construct an URL (because this information is for end users and our docs are more for technical people). For example, we have a section about linking to cbioportal here: https://www.cbioportal.org/webAPI, but I think maybe we should now have a separate page for this. Alternatively, we could put it in FAQ.

**Comment by jjgao** [comment]
- [] Let’s move the simple-filter to FAQ. 
- [] Please also expose the attribute ID on the study view interface, e.g. a column in Add Chart (hidden by default).

**Comment by zhx828** [comment]
@jjgao @inodb could you take another look? The second item from JJ's list will be in separate PR.

**Comment by jjgao** [comment]
@zhx828 I made some change to the FAQ section.

File: docs/study-view/priority.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Study View Customization with Priority Data
-Example of study view in public portal: https://www.cbioportal.org/study?id=acc_tcga,lgg_tcga#summary
+Example of study view in public portal: http://www.cbioportal.org/study?id=acc_tcga,lgg_tcga#summary
```
Comment (jjgao) [inline docs/study-view/priority.md]: why switch back to http here?

Comment (zhx828) [reply docs/study-view/priority.md]: @jjgao probably an issue when I did the rebase. Changed back to https

File: docs/SUMMARY.md [hunk 3]
```diff
@@ -36,6 +36,8 @@
 ### 2.3 Customization 
 * [Customizing your cBioPortal Instance via portal.properties](Customizing-your-instance-of-cBioPortal.md)
 * [More portal.properties Settings](portal.properties-Reference.md)
+* [How does the page layout the charts](study-view/layout.md)
```
Comment (inodb) [inline docs/SUMMARY.md]: @zhx828 maybe better:	`How does the study view organize the charts`

Comment (inodb) [reply docs/SUMMARY.md]: layout is a noun, so either `lay out` as a verb but that meaning is slightly different

File: docs/study-view/layout.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+## How does the page layout the charts
```
Comment (inodb) [inline docs/study-view/layout.md]: same here

File: docs/study-view/layout.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+Study view page is fully responsive, it will try to fit more charts in the same height as many as possible base on the browser width.
```
Comment (inodb) [inline docs/study-view/layout.md]: *based

Comment (inodb) [reply docs/study-view/layout.md]: `it will try to fit as many charts as possible based on the browser's width and height`

**Comment by jjgao** [comment]
Not why this wasn't merged? @zhx828

**Comment by zhx828** [comment]
@jjgao i think at then, I hesitated to include it because the filters hasn't been finalized. But I think we could now.

**Comment by zhx828** [comment]
@kalletlak study view has been updated significantly since I wrote this. Do you think you can provide more insights on how this document can be updated?

**Pull request merged**